.github/workflows/ci.yaml

```
@@ -0,0 +1,158 @@
+
+      - name: Run Formal Verification
+        run: |
+          mkdir -p verification-results
+          
+          # Initialize counter for verified specs
+          verified_count=0
+          
+          for spec in src/test/kontrol/*.k; do
+            if [ -f "$spec" ]; then
+              echo "Verifying $spec..."
+              
```

yamllint

[error] trailing spaces

(trailing-spaces)